.github/workflows/unit-test.yml

```diff
@@ -45,7 +45,55 @@ jobs:
         npx lerna run build
 
     - name: Running the tests
-      run: npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"
+      run: |
+        npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache" \
+          --testPathIgnorePatterns="packages/date/"
+
+  unit-js-date:
+    name: JavaScript (Date functions)
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        timezone: ['EST', 'GMT', 'CET']
```

Comment:
How about adding JST (+9) to this? So we run tests on the far end of the positive timezones.

1. EST, en_US
2. GMT, en_GB
3. CET, fr_FR (?)
4. JST, ja_JP

Reply:
Also should include settings to cover this scenario: https://github.com/WordPress/gutenberg/issues/27251#issuecomment-737203798

Reply:
> 1. EST, en_US
> 2. GMT, en_GB
> 3. CET, fr_FR (?)
> 4. JST, ja_JP

BTW, I think I'd do an actual matrix for `timezone` and `locale`, so we get all 16 combinations, rather than just 4...

Reply:
Do we also need daylight saving time? :thinking:

Reply:
I'm not sure if that's possible though 🤔 I mean, yes that would be great to set DST on and off.

Reply:
I think DST has to be handled by the tests themselves, since it's a matter of checking for the date at two different 
points in the year.

Reply:
You are right!